pkg/core/commands/deployer_create.go

```diff
@@ -282,6 +292,7 @@ and ` + cli.EnvFromFlagName + ` to map values from a ConfigMap or Secret.
 	cmd.Flags().BoolVar(&opts.Tail, cli.StripDash(cli.TailFlagName), false, "watch deployer logs")
 	cmd.Flags().StringVar(&opts.WaitTimeout, cli.StripDash(cli.WaitTimeoutFlagName), "10m", "`duration` to wait for the deployer to become ready when watching logs")
 	cmd.Flags().BoolVar(&opts.DryRun, cli.StripDash(cli.DryRunFlagName), false, "print kubernetes resources to stdout rather than apply them to the cluster, messages normally on stdout will be sent to stderr")
+	cmd.Flags().Int32Var(&opts.ContainerPort, cli.StripDash(cli.ContainerPortFlagName), 0, "`port` to be set as containerPort on the first container of the Deployment")
```

Comment:
This description is needlessly deep and not very informative. "Deployments" are an implementation detail and there is only container.

```suggestion
	cmd.Flags().Int32Var(&opts.ContainerPort, cli.StripDash(cli.ContainerPortFlagName), 0, "`port` the workload listens to for traffic. The value is exposed to the workload as the PORT environment variable")
```